Cookbook/Technical-Documents/RubyMine.md

```diff
@@ -0,0 +1,21 @@
+# RubyMine
+
+If you have to work with Ruby, i.e. to update our fastlane setup, you better use an IDE that comes with some quality of life tools other than syntax higlighting.
+We are extensively using JetBrains products in the company so our license includes such Ruby IDE as RubyMine.
+
+To install it go to https://www.jetbrains.com/ruby/download/#section=mac to download the macOS app.
+When asked about license enter the address of our license service. The service address is in 1Password. This service is only available from office network or via VPN.
+
+![](Assets/RubyMineLicense.png)
+
+Then open the root folder of our project. As most of the files are not in Ruby IDE will spend a lot of time tring to index files we don't need.
+To avoid that go to **Preferences -> Project Structure** and select all the foldres except `fastlane` folder as **excluded**.
```

Comment:
```suggestion
To avoid that, go to **Preferences -> Project Structure** and select all the folders except `fastlane` as **excluded**.
```

Reply:
Quick question, wouldn't it be easier to just open the `fastlane/` folder in the IDE, instead of opening the project's root folder and excluding everything but `fastlane/`?

Reply:
it won't recognise installed gems because Gemfile is not in the current directory